ci/pipeline/determine_tests_to_run.py

```diff
@@ -208,6 +208,15 @@ def get_commit_range():
                 # we pass, as the flag RAY_CI_DOC_AFFECTED is only
                 # used to indicate that tests/examples should be run
                 # (documentation will be built always)
+            elif changed_file.startswith("release/"):
+                if changed_file.startswith("release/ray_release"):
+                    # Release test unit tests are ALWAYS RUN, so pass
+                    pass
```

Comment:
Out of curiosity, where is this logic being handled?

Reply:
what do you mean? the env vars?

That's in https://github.com/ray-project/buildkite-ci-pipelines/tree/main/ray_ci

Reply:
The logic for "Release test unit tests are ALWAYS RUN"

Reply:
They have condition `ALWAYS` in the pipeline definition, and that logic is handled in the pipelines repo above